features/support/env.rb

```diff
@@ -113,6 +113,11 @@
 World(FactoryBot::Syntax::Methods)
 World(Warden::Test::Helpers)
 
+After do |scenario|
+  name = scenario.location.file.gsub('features/', '').gsub(%r{/\.|\/}, '-')
+  screenshot_image(name) if scenario.failed?
```

Comment:
Maybe wrap the name variable in the if scenario.failed? as well e.g. 
if scenario.failed?
name = ..
screenshot_image(name)
end

Reply:
Good catch! Fixed in next commit.